### PR TITLE
posts: first Cockpit-files release

### PR DESCRIPTION
**Pull request by jelly**
Preview: https://jelly.github.io/cockpit-project.github.io/blog/first-cockpit-files-release.html

### CODE REVIEW

File: _posts/2024-06-10-first-cockpit-files-release.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+![Cockpit Files list view](/images/cockpit-files-1-listview.png)
+
+## Upload support
+
+![Cockpit Files upload](/images/cockpit-files-1-upload.png)
+
+## Try it out
+
+We are working on getting Cockpit-files packaged in Fedora, Arch Linux and Debian. In the meantime this [copr repository](https://copr.fedorainfracloud.org/coprs/g/cockpit/main-builds/package/cockpit-files/) can be used to try out development snapshots, issues and feature requests can be reported on our [issue tracker](https://github.com/cockpit-project/cockpit-files/issues).
```
Comment (jelly) [inline _posts/2024-06-10-first-cockpit-files-release.md:27]: @martinpitt taken the liberty to mention Debian but of course I won't be doing this, but will you? :)

File: _posts/2024-06-10-first-cockpit-files-release.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+This first release includes download, upload, copy & paste, renaming of files & directories, directory creation and editing of permissions. In future releases we intend to add a basic file editor for configuration files, cut & paste, SELinux, ACL's, symbolic link creation support and more.
+
+## Card view
+
+![Cockpit Files overview](/images/cockpit-files-1-overview.png)
+
+## List view
+
+![Cockpit Files list view](/images/cockpit-files-1-listview.png)
```
Comment (allisonkarlitskaya) [inline _posts/2024-06-10-first-cockpit-files-release.md:19]: In general I think the screenshots are too large.  We might fake the contents of a "typical home directory" or other folder with 6-10 items in it which will show off all of the features (a folder or two, a wide range of file sizes, dates, file types) in a smaller amount of space.  An endless sea of folders with 'techy' names isn't too impressive looking.

Maybe also different folders/content for the icon vs list views to highlight the relative strengths of each?

File: _posts/2024-06-10-first-cockpit-files-release.md (hunk 1 above)
Comment (allisonkarlitskaya) [inline _posts/2024-06-10-first-cockpit-files-release.md:27]: Please either "Cockpit Files" or "cockpit-files".

Comment (allisonkarlitskaya) [inline _posts/2024-06-10-first-cockpit-files-release.md:27]: oxford comma, and Debian, please.

Comment (allisonkarlitskaya) [inline _posts/2024-06-10-first-cockpit-files-release.md:27]: comma splice: development snapshots, issues and feature requests

and maybe change "issues" to "bug reports"

File: _posts/2024-06-10-first-cockpit-files-release.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+---
+title: First Cockpit-Files release
```
Comment (allisonkarlitskaya) [inline _posts/2024-06-10-first-cockpit-files-release.md:2]: as above, "cockpit-files" or "Cockpit Files"

File: _posts/2024-06-10-first-cockpit-files-release.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+author: jelle
+date: 2024-06-10
+category: release
+tags: cockpit files manager browser
+---
+
+Last Friday we made the first release of cockpit-files, a new plugin for Cockpit to manage files on your server. Development on this new plugin was started by our GSoC student [Mahmoud Hamdy](https://github.com/MahmoudHamdy02) with the intention to replace existing [cockpit-navigator](https://github.com/45drives/cockpit-navigator/) plugin from 45Drives with a modern, automated tested and Cockpit styled plugin.
```
Comment (allisonkarlitskaya) [inline _posts/2024-06-10-first-cockpit-files-release.md:9]: Cockpit-styled with a hyphen.  But maybe mention Patternfly?  Is that a thing we do?  I notice that it's mentioned a lot when talking about the Anaconda webui...

**Review by allisonkarlitskaya — CHANGES_REQUESTED** [review]

**Comment by garrett** [comment]
I've made some changes, which crop the screenshots and add a screenshot style (to get a border, as well as added the ability to make them align to the left) and shuffled some content around so it's not just a wall of text.

Shift-reload the preview URL to see the changes, otherwise the images might be an older cached version.

File: _posts/2024-06-10-first-cockpit-files-release.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
+---
+title: "Cockpit Files: the first release"
+author: jelle
+date: 2024-06-10
+category: release
+tags: cockpit files manager browser
+---
+
+Last Friday, we released the first version of Cockpit Files, a plugin for Cockpit to manage files on your server through your web browser. Cockpit Files was initially started by Google Summer of Code (GSoC) student [Mahmoud Hamdy](https://github.com/MahmoudHamdy02) and is now under active development by the Cockpit team. The goal is to replace the functionality of the [cockpit-navigator](https://github.com/45drives/cockpit-navigator/) plugin from 45Drives and include automated testing per commit, a standard PatternFly-based interface, and consistency with the rest of Cockpit.
```
Comment (allisonkarlitskaya) [inline _posts/2024-06-10-first-cockpit-files-release.md:9]: Change the second , to a :.

**Review by allisonkarlitskaya — CHANGES_REQUESTED** [review]
The "0 B" files in the screenshots look implausible.  Can you please give those reasonable file sizes for the respective file types?

**Review by allisonkarlitskaya — APPROVED** [review]
plz squash when merging

**Comment by garrett** [comment]
> plz squash when merging

(There was an intentional separate commit that was unrelated to the other commit that was blog post... that was supposed to be separate. :frowning: )

**Pull request merged**